.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2023/06 to 2023/07
+1. Enhancement,AIRAC(2307) - Add Farnborough (EGLF) Apron B - Thanks to @rishab-alt
```

Comment:
```suggestion
1. AIRAC (2307) - Add Farnborough (EGLF) Apron B - Thanks to @rishab-alt
```
Is this actually an enhancement and AIRAC change? Just adding Apron B I think is AIRAC only :)